src/v2/api/controllers/TeacherController.ts

```diff
@@ -472,4 +472,24 @@ export class TeacherController {
   ) {
     return this.teacherService.connectTeacherWithCathedra(teacherId, cathedraId);
   }
+
+  @Delete('/:teacherId/cathedra/:cathedraId')
```

Comment:
The endpoint needs to be protected, create a permission and add it to ```@ApiEndpoint()```. See examples